Lib/fontTools/misc/psCharStrings.py

```diff
@@ -1002,25 +1002,23 @@ def compile(self, isCFF2=False):
 		encodeInt = self.getIntEncoder()
 		encodeFixed = self.getFixedEncoder()
 		i = 0
-		end = len(program)
-		while i < end:
+		while i < len(program):
```

Comment:
Any reason for this? Sure it's correct, but also calls `len(program)` more than is needed if `len(program)` doesn't change.

Reply:
brain fart; shows my inexperience using `while`